Code/Framework/AzFramework/AzFramework/Physics/HeightfieldProviderBus.h

```diff
@@ -26,17 +26,31 @@ namespace Physics
 
     struct HeightMaterialPoint
     {
+        static void Reflect(AZ::ReflectContext* context);
+
+        AZ_RTTI(HeightMaterialPoint, "{DF167ED4-24E6-4F7B-8AB7-42622F7DBAD3}");
```

Comment:
Jenkins run failed with `error: destructor called on non-final 'Physics::HeightMaterialPoint' that has virtual functions but non-virtual destructor`. It'll be fixed by adding a virtual destructor to `HeightMaterialPoint`.

Reply:
Fixed.